src/pkjs/index.js

```diff
@@ -24,11 +23,20 @@ Pebble.addEventListener("appmessage", function(e) {
 	if (Object.keys(buffer).length >= config.bufferLength) {
         const req = new XMLHttpRequest();
         req.open("POST", config.apiEndpoint);
+        req.responseType = "json";
         req.onload = function() {
-            console.log(this.status);
+            console.log("Server response: " + this.status + " - " + (this.status === 200));
+            if (this.status === 200 && this.response.count !== undefined) {
```

Comment:
With some later changes, this.response.count should always be !== undefined. But we're  leaving this check here just in case